### PR TITLE
Fix tab autocompletion

### PR DESCRIPTION
**Pull request by 12pm**

### CODE REVIEW

File: src/test/str.cpp [hunk 1]
```diff
@@ -72,8 +72,8 @@ TEST(Str, Utf8ToLower)
 	EXPECT_TRUE(str_utf8_comp_nocase(a, b) > 0);
 	EXPECT_TRUE(str_utf8_comp_nocase(b, a) < 0);
 
-	EXPECT_TRUE(str_utf8_comp_nocase_num("ÖlÜ", "ölüa", 3) == 0);
-	EXPECT_TRUE(str_utf8_comp_nocase_num("ÖlÜ", "ölüa", 4) != 0);
+	EXPECT_TRUE(str_utf8_comp_nocase_num("ÖlÜ", "ölüa", 5) == 0);
+	EXPECT_TRUE(str_utf8_comp_nocase_num("ÖlÜ", "ölüa", 6) != 0);
```
Comment (def-) [inline src/test/str.cpp]: That destroys the intention of the test.  It's supposed to check the first 3 characters isn't it?  Or is it now the number of bytes?

Comment (12pm) [reply src/test/str.cpp]: Says bytes here
https://github.com/ddnet/ddnet/blob/7da40ed1eabbc31b5414dc121d93e6f7bccb1627/src/base/system.h#L1665
and characters here
https://github.com/ddnet/ddnet/blob/7da40ed1eabbc31b5414dc121d93e6f7bccb1627/src/base/system.h#L1670

Comment (12pm) [reply src/test/str.cpp]: Bytes is correct currently

Comment (def-) [reply src/test/str.cpp]: Thanks, fixed.

**Pull request merged**